x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_table.test.tsx

```diff
@@ -32,113 +31,112 @@ describe('EnginesTable', () => {
     } as EngineDetails,
   ];
   const pagination = {
-    totalEngines: 50,
     pageIndex: 0,
-    onPaginate,
+    pageSize: 10,
+    totalItemCount: 50,
+    hidePerPageOptions: true,
   };
   const props = {
-    data,
+    items: data,
+    loading: false,
     pagination,
+    onChange,
   };
 
-  const wrapper = mountWithIntl(<EnginesTable {...props} />);
-  const table = wrapper.find(EuiBasicTable);
+  describe('basic table', () => {
+    const wrapper = mountWithIntl(<EnginesTable {...props} />);
```

Comment:
I added a new `describe` block here so that other blocks that needed to define/mount a new `wrapper` didn't have to come up with new `wrapper` names